### DISCUSSION TITLE
After 9.4.0 grafana monitoring template file where to download

### DISCUSSION THREAD

**Discussion by muarine** [apache/skywalking#10968] [Q&A] [answered]
Demo page monitor panel configuration grafana. Json template file where can I find
![image](https://github.com/apache/skywalking/assets/3285738/570322d4-a42f-4deb-a4ff-43dfe575c433)
I hope can get help, thank you

**Comment by wu-sheng** [comment, marked as answer]
First of all, PromQL doesn't mean we would have an official dashboard on Grafana. PromQL could do many integration things, more than Grafana.
And if you mean the preview on the demo, that is a showcase only, https://github.com/apache/skywalking-showcase/tree/main/deploy/platform/docker/config/promql

**Reply by muarine** [reply]
okay, thx
